clients/java/src/test/java/marquez/client/MarquezWriteOnlyClientTest.java

```diff
@@ -98,26 +97,26 @@ public void testMarkRunAs() throws UnsupportedEncodingException {
     Instant at = Instant.now();
     String atParam = URLEncoder.encode(String.valueOf(at), UTF_8.name());
     client.markRunAsRunning(runId, at);
-    verify(backend, times(1)).post("/api/v1/jobs/runs/" + runId + "/start?at=" + atParam);
+    verify(backend, times(1)).post("api/v1/jobs/runs/" + runId + "/start?at=" + atParam, null);
     client.markRunAsAborted(runId, at);
-    verify(backend, times(1)).post("/api/v1/jobs/runs/" + runId + "/abort?at=" + atParam);
+    verify(backend, times(1)).post("api/v1/jobs/runs/" + runId + "/abort?at=" + atParam, null);
     client.markRunAsCompleted(runId, at);
-    verify(backend, times(1)).post("/api/v1/jobs/runs/" + runId + "/complete?at=" + atParam);
+    verify(backend, times(1)).post("api/v1/jobs/runs/" + runId + "/complete?at=" + atParam, null);
     client.markRunAsFailed(runId, at);
-    verify(backend, times(1)).post("/api/v1/jobs/runs/" + runId + "/fail?at=" + atParam);
+    verify(backend, times(1)).post("api/v1/jobs/runs/" + runId + "/fail?at=" + atParam, null);
```

Comment:
I'm not sure I understand the usage of `null` being passed to `path()`. There is a method that already handle such case, see [`Backend.post(path)`](https://github.com/MarquezProject/marquez/blob/main/clients/java/src/main/java/marquez/client/Backend.java#L14)

Reply:
Done